### DISCUSSION TITLE
Input suggestions

### DISCUSSION THREAD

**Discussion by Gold240sx** [cosscom/coss#25] [Suggestions]
** Timestamp input **

Please build a video timestamp input to select a timestamp from a video (such as YouTube) with a max value (the value would be a parameter as the video length, optional). HH/MM/SS. 

** Number input with buttons Accessibility **

The number input with Chevrons / number input with plus and minus buttons are great but can be difficult for users who need accessibility assistance. The buttons should at least have an option of being bigger to make it easier to change the number. 

** Vertical Stepper **
(For multistep forms on mobile)

A vertical stepper bar which would be position fixed, with a scrollable body to the right for each step contents of the form. It would have numbers and bars between the numbers to indicate which step they were on. When clicked on would go to the step. 
It would be interesting on how to/ where to place the labels. 

Thanks!

**Comment by davidepacilio** [comment]
Hey @Gold240sx, thanks for contributing to the discussion! Could you please share a screenshot or image for each input you've suggested? This would really help us quickly understand and implement your ideas. Thanks again!

**Comment by Gold240sx** [comment]
Something like this...  I really wasn't exactly sure how to do the timestamp one, but I think it'd be quite close to your current time inputs, just again meant to input a time associated with a video / song... Ohh, I forgot to add the video duration to that one too... Hope this helps.

<a href="https://ibb.co/bXynyhG"><img src="https://i.ibb.co/DQ6n6Xj/SCR-20241031-dxew.png" alt="SCR-20241031-dxew" border="0" /></a>

**Comment by pasqualevitiello** [comment]
Hey @Gold240sx , thanks for the feedback. About the vertical stepper - it actually doesn't seem to be tied to inputs specifically - I mean, you can put any kind of content to the right of the stepper. Because of this, I think we should probably put this component in a different category. And for the timestamp input, what do you think about something like what's shown in the video below?

https://github.com/user-attachments/assets/d6d2bf38-9ec2-4c9d-a341-fc3ff0cd41ae

**Reply by Gold240sx** [reply]
Yeah that looks great! I still think an internal label may be easier to understand for the less tech savvy, but that is the component I was looking for.

**Comment by pasqualevitiello** [comment]
Cool, will add it next week!

**Comment by pasqualevitiello** [comment]
Added!

**Comment by Gold240sx** [comment]
The timestamp loos great! Nice job!

From: Pasquale Vitiello ***@***.***>
Date: Tuesday, November 5, 2024 at 2:35 AM
To: origin-space/originui ***@***.***>
Cc: Michael Martell ***@***.***>, Mention ***@***.***>
Subject: Re: [origin-space/originui] Input suggestions (Discussion #25)

Added!
